files-community/Files
# Code Quality: Removed FileTagsHelper & FileTagsManager

## Summary

As the title. There's some changes on how to store file tags as a list for Sidebar items, so please validate changes first.


## Steps To Validate Changes

1. Launch the app
2. Go to Home
3. Sidebar won't crash
4. Home won't crach
5. See FileTags widget shows the same content as the main branch

## PR Checklist

- [ ] **Development**: _N/A_

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
